tests/main/sbuild/task.yaml

```diff
@@ -29,6 +29,10 @@ restore: |
     rm -f /etc/schroot/chroot.d/sid-amd64-sbuild-*
 
 debug: |
+    # There there's a log file and a symbolic link pointing to it.
+    # The non-symlink has a time-stamp and we can match on the "Z" timezone
+    # marker to find it.
+    tail -n 100 *Z.build
```

Comment:
and if there isn't a log file?

Reply:
Then the test will fail anyway.

Reply:
yes, but so will the debug, and then you can't `-debug`?

Reply:
Do you mean that this would prevent a `-debug` shell?